elyra/metadata/schemas/local-directory-catalog.json

```diff
@@ -39,12 +39,11 @@
           "description": "Description of this Component Catalog",
           "type": "string"
         },
-        "runtime": {
-          "title": "Runtime",
-          "description": "The runtime associated with this Component Catalog",
+        "runtime_type": {
+          "title": "Runtime Type",
+          "description": "The type of runtime associated with this Component Catalog",
           "type": "string",
-          "$comment": "This enum is dynamically generated to contain the available runtime values.",
-          "enum": ["{currently-configured-runtimes}"],
+          "enum": ["KUBEFLOW_PIPELINES", "APACHE_AIRFLOW"],
```

Comment:
This requires updates to schemas (elyra code) every time you add external runtime (e.g. ARGO). 
How about making a runtime error that the runtime type was not found?

Reply:
The idea was that these are explicit because each runtime type must be qualified.  Once qualified the new type can be added.  This is only necessary for new _types_ not external implementations of an existing type.

If necessary, we could have the schemas provider do the filtering again, but that would be to on _trim_ the contents relative to what is active (installed), with the base schema indicating what has been qualified.

We can visit this once we no longer have our two out-of-the-box processors unconditionally available.  However, I believe the enum itself (prior to runtime) should be a reflection of support/qualification.